setup

```diff
@@ -15,16 +15,13 @@ INSTALLDIR="$1"
 ROOTFS="$2"
 [ ! -z "${ROOTFS}" ] || ROOTFS=/usr/lib/qm/rootfs
 
-. /etc/os-release
-
-systemctl stop qm 2> /dev/null
+systemctl stop qm.service 2>/dev/null || true
 
 install() {
     rootfs=$1
```

Comment:
shouldn't this be $2

Reply:
No it is $2 for the script but this is passed in as $1 in the function.